client_plugin/drivers/shared/kvstore/etcdops/etcdops.go

```diff
@@ -59,6 +60,7 @@ const (
 	etcdClusterStateExisting = "existing"
 	requestTimeout           = 5 * time.Second
 	checkSleepDuration       = time.Second
+	gcTicker                 = 5 * time.Second
```

Comment:
Isn't this too expensive to run every 5 seconds? Maybe 30 seconds gap should be fine.